kyakusahmed/EPIC-EMAIL
# Ft get messages #165085272

### What does this PR do?
get all messages inbox

### Description of Task to be completed?
Using java script fetch, a user can now successfully view all his inbox.

### How should this be manually tested?
To test if the user is able to view his inbox,  on log in , he should be directed straight ti his inbox

Important Notes
 [gh-pages](https://kyakusahmed.github.io/EPIC-EMAIL/templates/signup.html)
@bekeplar  @jenny @neelxie @nanfuka @Wabuluka
